framework/base/ErrorHandler.php

```diff
@@ -116,11 +116,8 @@ public function handleException($exception)
                 }
                 exit(1);
             }
-        } catch (\Exception $e) {
-            // an other exception could be thrown while displaying the exception
-            $this->handleFallbackExceptionMessage($e, $exception);
         } catch (\Throwable $e) {
-            // additional check for \Throwable introduced in PHP 7
+            // additional check for \Throwable
```

Comment:
Since the minium requirement is 7.1 how about

`catch(Exception|Throwable $e)`